tox.ini

```diff
@@ -15,6 +15,7 @@ deps =
 [testenv:test]
 deps = {[base]deps}
 commands =
+    pip install sqlalchemy
```

Comment:
why we need install sqlalchemy here since we have already put it in requirements?

Reply:
Forgot to remove it, note it